internal/graph/cached_resolver.go

```diff
@@ -57,14 +56,6 @@ var _ CheckResolver = (*CachedCheckResolver)(nil)
 // instance.
 type CachedCheckResolverOpt func(*CachedCheckResolver)
 
-// WithMaxCacheSize sets the maximum size of the Check resolution cache. After this
-// maximum size is met, then cache keys will start being evicted with an LRU policy.
-func WithMaxCacheSize(size int64) CachedCheckResolverOpt {
```

Comment:
FYI this is never referenced, except in tests, which can make do with the default.